native/packages/react-native-bpk-component-picker/src/BpkPickerMenu.android.js

```diff
@@ -126,6 +126,7 @@ const BpkPickerMenu = (props: PickerMenuProps) => {
             return (
               <BpkPickerItem
                 {...item}
+                allowFontScaling={false}
```

Comment:
I think this is in the wrong place, no? It should be on the button link component.